app/models/game.rb

```diff
@@ -4,4 +4,38 @@ class Game < ApplicationRecord
   has_one :black_player, class_name: 'User', dependent: false
   has_one :winning_player, class_name: 'User', dependent: false
   has_many :pieces, dependent: false
+
+  attr_accessor :board_positions
+
+  def initialize
+    @board_positions = Array.new(8) { Array.new(8, ' ') }
+  end
```

Comment:
Can remove this. Will use the database instead. 😄

Reply:
@mscoutermarsh Hmm, how so?

Reply:
Rook.create(x_position: 0, y_position: 0, game: self, color: 'black')

Reply:
Yep, realize now that was a silly question 😂